includes/class-freemius.php

```diff
@@ -516,7 +567,11 @@ private function _find_caller_plugin_file() {
 				}
 			}
 
-			$plugin_file = fs_find_caller_plugin_file();
+			if ( $this->_caller_is_plugin() ) {
+				$plugin_file = fs_find_caller_plugin_file();
+			} else {
+				$plugin_file = dirname( WP_FS__DIR ) . '/functions.php';
```

Comment:
@fajardoleo I think this code is incorrect for cases when the newest SDK is running from a plugin, but currently, we are running it from a theme. The reason it will happen is because `WP_FS__DIR` is a reference to the SDK location, not to the context plugin/theme.